pkg/watcher/template_watch.go

```diff
@@ -16,16 +17,18 @@ func (t *fsnotifyTemplateWatcher) Run() (chan []byte, chan error) {
 
 func (t *fsnotifyTemplateWatcher) watch(updates chan []byte, errors chan error) {
 	for ev := range t.watcher.Events {
-		glog.V(6).Infof("observed %s event on %s", ev.String(), ev.Name)
+		if ev.Op&fsnotify.Write == fsnotify.Write {
```

Comment:
I'm not sure if this is an edge case that needs consideration, but should we also respect `fsnotify.Create` events?

```suggestion
		if ev.Op&(fsnotify.Write|fsnotify.Create) > 0 {
```